components/Blog/BlogCard/index.tsx

```diff
@@ -0,0 +1,46 @@
+import { FormattedDate, FormattedMessage } from 'react-intl';
+import styles from './index.module.scss';
+import LocalizedLink from '../../LocalizedLink';
+import navigation from '../../../navigation.json';
+import type { BlogPost } from '../../../types';
+import type { FC } from 'react';
+
+const getBlogCategoryUrl = (category: string): string =>
+  `${navigation.blog.link}/${category}/`;
+
+const getBlogPostUrl = (slug: string) =>
+  slug.endsWith('/') ? slug : `${slug}/`;
```

Comment:
AFAIK we don't use trailing slashes on links here, so no need to have this function